### ISSUE TITLE
Plan the 3.7 Release

### ISSUE THREAD

**Issue by Suremaker** [nunit/nunit#2167]
Hello,

I would like to ask what is the planned release date for nunit 3.7? I cannot find anything in the documentation and [the 3.7 milestone](https://github.com/nunit/nunit/milestone/25) shows past 1 month.

I am asking because I'm planning to release a new version of my project and while there are few open issues that depends on upcoming nunit 3.7, I'd like to know if there is a chance to get them unblocked soon, or rather I should park them for a longer time.

Thanks

**Comment by rprouse** [comment]
I was really hoping to get the port to .NET Standard 1.3 into the 3.7 release, but I keep running into problems with my work on that.

@nunit/core-team what do you think? Should we push 3.7 out soon? We are well past our milestone. I was okay with that because we released 3.6.1 close to the original milestone, but we are really starting to slip.

If we release 3.7, are there any outstanding issues or pull requests that people are working on that we should wait for? I would like to take a look at #2165.

**Comment by CharliePoole** [comment]
I think we should get the release out. It's always possible to do another release when there is something important.

**Comment by jnm2** [comment]
I think sooner is always better. It would be nice for https://github.com/nunit/nunit/pull/2023 to get reviewed because I think it's ready to go, unless we have a reason to postpone it.

**Comment by ChrisMaddock** [comment]
Agreed. The .NET Standard 1.3 port shouldn't block anything I don't think, as PCL can still be used - but it would be good to get test case parallelisation out. 🙂 

Agreed we should wait on #2165 - that's verging on critical. I was planning to get #2152 finished this weekend, and would like to get that in, so I can get runner support in too, without a three month delay.

From a quick scan, I think we should also try and get merged: #1855, #2023, #2041. Would be great if someone else can review #2023, I've been ignoring it, as my thread management good-practice knowledge is minimal!

**Comment by CharliePoole** [comment]
I also have not felt I know enough about message loops these days to review #2023. Last time I dealt with them was in Windows apps written in C.

**Comment by jnm2** [comment]
@CharliePoole Would it help if I better documented the code?

**Comment by CharliePoole** [comment]
Well, in my case it probably would. :smile:

**Comment by rprouse** [comment]
I've moved everything that was mentioned into the 3.7 milestone so we can track them easily. I am also going to set the date for the 3.7 release to next Friday. Let's try to get everything done by then and I will release on Friday or Saturday if everything works out.

**Comment by rprouse** [comment]
I am going to close this as I think everything is lined up for a release this weekend.
